osmaxx/excerptexport/models/output_file.py

```diff
@@ -21,6 +22,16 @@ class OutputFile(models.Model):
     public_identifier = models.UUIDField(primary_key=False, default=uuid.uuid4, verbose_name=_('public identifier'))
     export = models.OneToOneField(Export, related_name='output_file', verbose_name=_('export'))
 
+    def __str__(self):
+        return \
+            '[' + str(self.id) + '] ' \
+            + ('file: ' + os.path.basename(self.file.name) + ', ' if (self.file and self.file.name) else '') \
+            + 'identifier: ' + str(self.public_identifier)
```

Comment:
Never use a template in `__str__`, if you get an error, it will not even be able to display the representation of the class it has.

I just moved this line up, where it felt more appropriate, haven't changed anything. This always was a bad string representation for this class, but should not be part of this pull-request. You might want to consider creating an issue for this :wink: